src/hydra/fetchJsonLd.js

```diff
@@ -16,6 +16,11 @@ export default function fetchJsonLd(url, options = {}) {
     options.headers.set('Accept', jsonLdMimeType);
   }
 
+  if (null === options.headers.get('Authorization')) {
```

Comment:
It does not satisfy the needs just using the decorator. (or I don't have the good version atm).